Resources/config/mongodb.xml

```diff
@@ -136,7 +136,11 @@
             <argument type="tagged" tag="doctrine_mongodb.odm.command_logger" />
         </service>
 
-        <service id="doctrine_mongodb.odm.command_logger" class="Doctrine\Bundle\MongoDBBundle\APM\PSRCommandLogger" public="false">
+        <service id="doctrine_mongodb.odm.stopwatch_command_logger" class="Doctrine\Bundle\MongoDBBundle\APM\StopwatchCommandLogger" public="false">
+            <argument type="service" id="debug.stopwatch" on-invalid="null" />
+        </service>
+
+        <service id="doctrine_mongodb.odm.psr_command_logger" class="Doctrine\Bundle\MongoDBBundle\APM\PSRCommandLogger" public="false">
```

Comment:
We can't change service names just like that as it breaks backward compatibility. The service should still be available under it's old name but marked as deprecated, like described in https://symfony.com/blog/new-in-symfony-4-3-deprecating-service-aliases

Reply:
Added an alias and marked it as deprecated.